models/UserModel.js

```diff
@@ -1,17 +1,92 @@
-const mongoose = require('mongoose');
-const Schema = mongoose.Schema;
+const mongoose = require('mongoose'), 
+      bcrypt = require('bcryptjs'),
+      Schema = mongoose.Schema,
+      SALT_WORK_FACTOR = 10;
 
-const userSchema = mongoose.Schema({
+const UserSchema = mongoose.Schema({
     firstName: {type: String, required: true},
```

Comment:
Please add trim: true to firstName, lastName,and email.

Reply:
done :)